release/README.DOCKER.md

```diff
@@ -1,36 +1,35 @@
-# BUX Wallet Frontend
+# SPV Wallet Frontend
 
-The `bux-wallet-frontend` is a referential frontend application for a custodial web wallet. 
-It is used in conjunction with the `bux-wallet-backend` component, which serves as the backend server. The frontend is written in React and can be run as a Docker image named `${DOCKERHUB_OWNER}/${DOCKERHUB_REPO}`, exposed on port 3002.
+The `spv-wallet-web-frontend` is a referential frontend application for a custodial web wallet.
+It is used in conjunction with the `spv-wallet-web-backend` component, which serves as the backend server. The frontend is written in React and can be run as a Docker image named `${DOCKERHUB_OWNER}/${DOCKERHUB_REPO}`, exposed on port 3002.
 
 ## Running as a Docker Image
 
-To run the `bux-wallet-frontend` as a Docker image, execute the following command:
+To run the `spv-wallet-web-frontend` as a Docker image, execute the following command:
 
 ```bash
 docker run -p 3002:3002 -v /path/to/env-config.json:/usr/share/nginx/html/env-config.json ${DOCKERHUB_OWNER}/${DOCKERHUB_REPO}:latest
 ```
 
 Replace `/path/to/env-config.json` with the actual path to your `env-config.json` file.
 
-This command will run the `bux-wallet-frontend` Docker image, exposing it on port 3002. The mounted `env-config.json` file will provide the necessary configuration to the frontend application.
+This command will run the `spv-wallet-web-frontend` Docker image, exposing it on port 3002. The mounted `env-config.json` file will provide the necessary configuration to the frontend application.
 
 ## Configuration
 
-To configure the `bux-wallet-frontend` Docker image, you need to create and mount a file named `env-config.json` under the location `/usr/share/nginx/html/env-config.json` in the Docker container.
+To configure the `spv-wallet-web-frontend` Docker image, you need to create and mount a file named `env-config.json` under the location `/usr/share/nginx/html/env-config.json` in the Docker container.
```

Comment:
The part of this documentation about `docker-compose` & `start.sh` script, should be reviewed after we make adjustments to that files.

Reply:
Actually there is nothing about start.sh, it is about docker image only 🙂 
It is used as a description on dockerhub. And it seems ok.